tomato-timer/src/firebase.js

```diff
@@ -0,0 +1,11 @@
+import * as firebase from 'firebase';
+import {authkey} from './apikeys'
+export const config = {
+    apiKey: authkey,
+    authDomain: "tomato-timer-30220.firebaseapp.com",
+    databaseURL: "https://tomato-timer-30220.firebaseio.com/",
```

Comment:
I'd keep this URL in a separate config file or use environment variables as well, not as critical as an API key but generally good practice